### PR TITLE
feat(enumeration-widget): add multiple items in a row

### PR DESCRIPTION
**Pull request by talend-sromain**
**What is the problem this PR is trying to solve?**

Currently we can only add one by one items on the list

**What is the chosen solution to this problem?**

New action button to keep add another item + default action on enter to reset the field & prepare to add another

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/Enumeration/Header/HeaderInput.component.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames';
 import Action from '../../Actions/Action';
 import theme from './Header.scss';
 
-let inputRef;
+let internalInputRef;
```
Comment (acateland) [inline packages/components/src/Enumeration/Header/HeaderInput.component.js]: if this component get used more than once into another component this value will be shared between them.

File: output/components.eslint.txt [hunk 2]
```diff
@@ -13,6 +13,22 @@ Scoping to packages that match 'react-talend-components'
   2:1  error  '@kadira/react-storybook-addon-info' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
   7:2  error  Unexpected require()                                                                                      global-require
 
+/home/travis/build/Talend/ui/packages/components/src/Enumeration/Enumeration.component.js
+   94:2   error  Mixed spaces and tabs                       no-mixed-spaces-and-tabs
```
Comment (jare-talend) [inline output/components.eslint.txt]: plz fix the eslint errors

File: packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js [hunk 3]
```diff
@@ -626,8 +681,12 @@ class EnumerationWidget extends React.Component {
 		return (
 			<input
 				type="file"
-				ref={(element) => { this.inputFile = element; }}
-				onChange={(event) => { this.importFile(event); }}
+				ref={(element) => {
```
Comment (jare-talend) [inline packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js]: it's normal to reformat this. My eslint is good on this part.

File: packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js [hunk 4]
```diff
@@ -418,6 +430,39 @@ class EnumerationWidget extends React.Component {
...
+				this.addFailHandler.bind(this))
+		) {
+			this.setState({
+				headerInput: this.loadingInputsActions,
+			});
+			this.input.focus();
+		} else if (!this.valueAlreadyExist(value.value)) {
+			this.setState(
+				{
+					items: this.state.items.concat([{
```
Comment (acateland) [inline packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js]: using setState this wayn don't garanties that this.state.items is in intended state. see https://facebook.github.io/react/docs/react-component.html#setstate

File: packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js [hunk 5]
```diff
@@ -418,6 +430,39 @@ class EnumerationWidget extends React.Component {
...
+						values: this.parseStringValueToArray(value.value),
+						inputValue: '',
+					}]),
+				},
+				this.setFormData.bind(this)
```
Comment (acateland) [inline packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js]: this appear 5 time in the code and this should be bind into the constructor

File: output/components.eslint.txt (hunk 2 above)
Comment (talend-sromain) [inline output/components.eslint.txt]: oh yes, what have done my ide

File: packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js [hunk 6]
```diff
@@ -450,6 +468,34 @@ class EnumerationWidget extends React.Component {
...
+				this.constructor.parseStringValueToArray(value.value),
+				this.validateAndAddSuccessHandler.bind(this),
+				this.addFailHandler.bind(this))
+		) {
+			this.setState({
+				headerInput: this.loadingInputsActions,
+			});
+			this.input.focus();
+		} else if (!this.valueAlreadyExist(value.value)) {
+			const items = this.state.items.concat([{
```
Comment (acateland) [inline packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js]: same as before this.state my not be in intended shape please use following state update form
this.setState((prevState, props) => { items: prevState...retrieving prevstate data... , inputValue: ''})

**Pull request merged**